tests/resources/conformance_suites/nl_nt16/br_kvk/2-04-invalid-period.xbrl

```diff
@@ -59,5 +52,6 @@
         </period>
     </context>
     <jenv-bw2-i:RightsGrantedExercisePrice contextRef="ctx-2" xml:lang="en" decimals="2" unitRef="usd">100.00</jenv-bw2-i:RightsGrantedExercisePrice>
-    <jenv-bw2-i:ShareCapitalInTheCurrencyOfDenominationWhenDifferentPresentationCurrencyExchangeRateUsed contextRef="ctx-2" xml:lang="en" decimals="INF" unitRef="pure">1.1</jenv-bw2-i:ShareCapitalInTheCurrencyOfDenominationWhenDifferentPresentationCurrencyExchangeRateUsed>
+    <jenv-bw2-i:ShareCapitalInTheCurrencyOfDenominationWhenDifferentPresentationCurrencyExchangeRateUsed contextRef="ctx-2" xml:lang="en" decimals="INF" unitRef="pure">1.1
+    </jenv-bw2-i:ShareCapitalInTheCurrencyOfDenominationWhenDifferentPresentationCurrencyExchangeRateUsed>
```

Comment:
Out of curiosity, why add newlines to fact values?  It's the same PSVI value, but it seems odd to me.

Reply:
I ran IntelliJ's formatting across all of the template files (see other comment) just for consistency. I believe it does this when the line reaches a certain length. I was fine with it since it doesn't effect the testcase results, but I'd also be open to conforming to other formatting rules.

Reply:
Auto-formatters for test data seem unwise, particularly for data files (what if we needed to test that Arelle was handling whiteSpace=collapse for this concept?).

Reply:
That makes sense, I suppose for the initial template files I was more concerned with just having consistent formatting to start with. I've opted to just not include the formatting changes in this PR since I anticipate that these NL conformance suite files might be removed in favor of the mentioned template approach.